### PR TITLE
Updated to use the Jakarta installation of Nexus

### PR DESCRIPTION
**Pull request by tomjenkinson**
Resolves https://github.com/eclipse-ee4j/ee4j/issues/55

### CODE REVIEW

**Comment by ivargrimstad** [comment]
@tomjenkinson Please add sign-off with your email associated with your eclipse account to your commit

**Comment by kwsutter** [comment]
@tomjenkinson We need proper sign-off on this PR.  Did you accidentally use the wrong email address when you committed?

File: parent/pom.xml [hunk 1]
```diff
@@ -139,7 +139,7 @@
                     <version>1.6.8</version>
                     <configuration>
                         <serverId>ossrh</serverId>
-                        <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                        <nexusUrl>https://jakarta.oss.sonatype.org/</nexusUrl>
```
Comment (kwsutter) [inline parent/pom.xml]: Nit.  But, to be consistent with the previous usage in the pom, should these references be changed to properties as well?  We could have a property for the top level url and then re-use that for the sub-directory references.

File: parent/pom.xml [hunk 2]
```diff
@@ -247,7 +247,7 @@
                 <repository>
                     <id>sonatype-nexus-snapshots</id>
                     <name>Sonatype Nexus Snapshots</name>
-                    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (kwsutter) [inline parent/pom.xml]: Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

File: parent/pom.xml [hunk 3]
```diff
@@ -260,7 +260,7 @@
                 <pluginRepository>
                     <id>sonatype-nexus-snapshots</id>
                     <name>Sonatype Nexus Snapshots</name>
-                    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (kwsutter) [inline parent/pom.xml]: Same comment as above...  Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

File: parent/pom.xml [hunk 4]
```diff
@@ -284,7 +284,7 @@
                 <repository>
                     <id>sonatype-nexus-staging</id>
                     <name>Sonatype Nexus Staging</name>
-                    <url>https://oss.sonatype.org/content/repositories/staging</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/staging</url>
```
Comment (kwsutter) [inline parent/pom.xml]: Same comment as above... Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

File: parent/pom.xml [hunk 5]
```diff
@@ -297,7 +297,7 @@
                 <pluginRepository>
                     <id>sonatype-nexus-staging</id>
                     <name>Sonatype Nexus Staging</name>
-                    <url>https://oss.sonatype.org/content/repositories/staging</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/staging</url>
```
Comment (kwsutter) [inline parent/pom.xml]: Same comment as above... Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

File: parent/pom.xml (hunk 2 above)
Comment (bshannon) [reply parent/pom.xml]: +1, although note trailing slash, which may or may not matter.

File: parent/pom.xml (hunk 4 above)
Comment (bshannon) [reply parent/pom.xml]: This is "staging", not "snapshots".

But yes, why not define a property for consistency and use it.

**Comment by Tomas-Kraus** [comment]
@bshannon replace all urls with properties sounds like a good idea. We can do it in next cycle.

**Pull request merged**